contracts/colony/ColonyFunding.sol

```diff
@@ -106,6 +107,38 @@ contract ColonyFunding is
     emit ColonyFundsClaimed(msgSender(), _token, feeToPay, remainder);
   }
 
+  function claimDomainFunds(address _token, uint256 _domainId) public stoppable {
+    require(domainExists(_domainId), "colony-funding-domain-does-not-exist");
+    address domainTokenReceiverAddress = IColonyNetwork(colonyNetworkAddress)
```

Comment:
Thinking more about whether this deployment should go on the network or the colony, while putting it on the colony seems preferable to avoid the external call, if we're going to check the resolver each time then the external call will be inevitable, and if so probably best to keep all the deployment logic in one place.